build_sdk.py

```diff
@@ -242,6 +242,21 @@ class ConfigInfo:
             "KernelRiscvExtF": True,
         },
     ),
+    BoardInfo(
+        name="ariane",
+        arch=KernelArch.RISCV64,
+        gcc_cpu=None,
+        loader_link_address=0x90000000,
+        kernel_options={
+            "KernelIsMCS": True,
+            "KernelPlatform": "ariane",
+            "KernelRiscvExtD": True,
+            "KernelRiscvExtF": True,
+        },
+        examples={
+            "hello": Path("example/ariane/hello")
+        }
```

Comment:
You'll need to remove this to get the SDK to build (the way examples work has been changed since the PR got made).